test/extended/apiserver/graceful_termination.go

```diff
@@ -61,6 +62,45 @@ var _ = g.Describe("[sig-api-machinery][Feature:APIServer][Late]", func() {
 		}
 	})
 
+	g.It("kubelet terminates kube-apiserver gracefully extended", func() {
+		controlPlaneTopology, err := exutil.GetControlPlaneTopology(oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		if *controlPlaneTopology == configv1.ExternalTopologyMode {
+			g.Skip("oc must-gather doesn't currently support external controlPlaneTopology")
+		}
+
+		tempDir, err := ioutil.TempDir("", "test.oc-adm-must-gather.")
+		o.Expect(err).NotTo(o.HaveOccurred())
+		defer os.RemoveAll(tempDir)
+
+		var finalMessageBuilder strings.Builder
+		terminationRegexp := regexp.MustCompile(`Previous pod .* did not terminate gracefully`)
+		terminationLogsPerServer := gatherMustGatherFor(oc, tempDir, clihelpers.IsTerminationLog, extractAPIServerNameFromTerminationFile)
```

Comment:
Don't use must-gather for this.  If you just need a hostmount pod, use that.  Removing layers will help future us.

Reply:
What is your concern here? 

Please note that this test doesn't download the entire must-gather tarball. It uses must-gather to only collect the audit logs. While it is true that the tarball contains more files than this test needs, I should have given the function a more specific name.

An alternative approach would be to copy only the files we need. Starting new pods on each master, checking their state, and then copying the output doesn't seem simpler.

Another option could be to use `oc adm node-logs` to copy specific files. However, we would still need to identify all the nodes on which this command should be executed.

Reply:
I have opened a PR that doesn't use the must-gather tool for downloading audit logs - https://github.com/openshift/origin/pull/29015